File: python/svix/models/list_response_endpoint_out.py
# this file is @generated
import typing as t

from .common import BaseModel
from .endpoint_out import EndpointOut


class ListResponseEndpointOut(BaseModel):
    data: t.List[EndpointOut]

    done: bool

    iterator: t.Optional[str]

    prev_iterator: t.Optional[str] = None

File: kotlin/lib/src/main/kotlin/models/ListResponseEndpointOut.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class ListResponseEndpointOut(
    val data: List<EndpointOut>,
    val done: Boolean,
    val iterator: String? = null,
    val prevIterator: String? = null,
)
